images/linux/scripts/installers/mysql.sh

```diff
@@ -9,19 +9,31 @@ source $HELPER_SCRIPTS/document.sh
 
 export ACCEPT_EULA=Y
 
-# Install MySQL Client
-apt-get install mysql-client -y
+if isUbuntu16 || isUbuntu18 ; then
+    apt-get install mysql-client -y
+fi
 
-# InstallMySQL database development files
-apt-get install libmysqlclient-dev -y
+if isUbuntu20 ; then
+    # Install mysql 8 for Ubuntu 20.
 
-# Install MySQL Server
+    debconf-set-selections <<< 'mysql-apt-config mysql-apt-config/select-server select mysql-8.0'
```

Comment:
is this code so the install call below will default to the 8 instead of 5?

Reply:
Yes, it is. By default package configuration contains mysql version 5, that is why we need to manually specify version 8 in the script for ubuntu 20.04.